src/rpm/common_test.go

```diff
@@ -0,0 +1,181 @@
+package rpm
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func hex(b *bytes.Buffer) string {
+	return fmt.Sprintf("%x", b.Bytes())
+}
+
+func TestPackTag(t *testing.T) {
```

Comment:
To be honest I don't understand a way how to obtain "tested" values...
May be you could add a some schema how every value is calculated... or maybe a link to doc/spec.

Reply:
I've added a doc in `tagset.go` file.